### ISSUE TITLE
Not saved snippet

### ISSUE THREAD

**Issue by Cruising0904** [maaslalani/nap#49]
**Describe the bug**
![image](https://user-images.githubusercontent.com/29267295/232891211-cc9004e6-3ee4-4bbf-871c-3865e3621783.png)
I make snippet like this, after close and open nap again then it is not loaded 

In ~/.local/share/nap directory 
there are some files I previously created in nap which do not show in the application
![image](https://user-images.githubusercontent.com/29267295/232892552-102a2388-eca6-4f07-af14-b1988be16604.png)

also the application shows depends on directory 
In .config 
![image](https://user-images.githubusercontent.com/29267295/232892753-2f8883a1-fd5c-4b22-b715-1f522a354f08.png)
I never made Notes folder and FizzBuzz but I did usage 

In ~/.dotfiles which I put my config files here Nothing survive from here 
In addition I put nap binary here. 

Lastly $HOME directory 
![image](https://user-images.githubusercontent.com/29267295/232893503-9f1eab52-a95b-4da5-aa25-f4d50885a362.png)
It appears to show snippets just minute ago

**Expected behavior**
Whenever execute nap have the same data

**Desktop (please complete the following information):**
 - OS: Arch Linux 
 - Browser brave
 
I love this program  ❤️

**Comment by Cruising0904** [comment]
It seems like it does not support absolute path. it keeps generating relative path from where I execute nap

**Comment by jolheiser** [comment]
What does your config look like?

**Comment by Cruising0904** [comment]
> What does your config look like?
> 
> Also what version are you using?

Thanks for the quick answer.
I will let you know when I get home

**Comment by Cruising0904** [comment]
![image](https://user-images.githubusercontent.com/29267295/233812469-e6650344-53ba-4931-b7bf-a15ee6d8c397.png)
sorry for the late. 

I used default configuration if you need more...

**Comment by Cruising0904** [comment]
sorry for the late

> What does your config look like?
> my config was just default setting like github page

![image](https://user-images.githubusercontent.com/29267295/233812806-4a869896-dddc-4eba-917d-264b48818f71.png)


> Also what version are you using?
I installed with go 

I could not identified the version of application
![image](https://user-images.githubusercontent.com/29267295/233812899-2b1a3887-6834-4238-958f-561cbc0aa504.png)

and.. My env is like 
![image](https://user-images.githubusercontent.com/29267295/233812918-91917f0e-1831-4595-9452-db12476dcf24.png)

**Comment by jolheiser** [comment]
Note that the example uses `~` for home, not `$HOME`. 
Env vars are not expanded in the config afaik.

**Comment by Cruising0904** [comment]
> Note that the example uses `~` for home, not `$HOME`. 
> Env vars are not expanded in the config afaik.

The way I put $HOME becuse it created '~' in every directory I add nap. It was the same when I did with ~/.config/nap 
That is why I changed it.
And with or without it consequences are the same 😥 
I will upload again with the result of after change the config too!

**Comment by jolheiser** [comment]
Ah, actually I bet I know the issue. The tilde expansion isn't in a released version.

You would need an absolute path until then.

**Comment by Cruising0904** [comment]
Thanks~ will try this. I thought relative path is not working so I thought I put absolute path with $HOME this is not it haha I guess I have to put like /home/username/~ ?

**Comment by jolheiser** [comment]
> Thanks~ will try this. I thought relative path is not working so I thought I put absolute path with $HOME this is not it haha I guess I have to put like /home/username/~ ?

Right, `$HOME` isn't absolute, it would need to be expanded (which is probably a decent feature request 😉).
Using something like `/home/<user>/...` should work for the time being. 🙂

**Comment by Cruising0904** [comment]
Thanks for kind answer.  
Now I have tried editing config file, removing ~/.config/nap, ~/.local/share/nap , rebooting, reinstalling (after removing binary and go install github.com/maaslalani/nap@main)  not affected. What else should I look for?

**Comment by jolheiser** [comment]
If you are using `main` you should be able to use `~`. If not, let me know if your config has changed more.

**Comment by Cruising0904** [comment]
Sorry for the late reply...

However that is not the point. The app create hidden folder where the command executed respectively.
call nap from home directory it creates its json file in hidden folder save snippets in the file
after call nap from go directory it also creates its json file in its hidden folder saves snippets in the file 
there will be ~/$HOME/.nap/snippets.json, go/$HOME/.nap/snippets.json
In Consequences I might forget the contents  after while I forget where I called it from.. 
Hopefully I made it clear..
